server/src/main/resources/webapp/scripts/components/auth/provider/auth.session.service.js

```diff
@@ -12,9 +12,13 @@ angular.module('CentralDogmaAdmin')
                      headers: {
                        'Content-Type': 'application/x-www-form-urlencoded'
                      }
-                   }).then(function (sessionId) {
-                     if (sessionId !== null) {
-                       $window.localStorage.setItem('sessionId', sessionId);
+                   }).then(function (accessToken) {
+                     var sessionId;
```

Comment:
I'm not sure, but I think we need to check `access_token` property. How about this?
```
 if (accessToken === null ||
     Angular.isUndefined(accessToken.access_token) ||
     accessToken.access_token === null) {
   return null;
 }
 $window.localStorage.setItem('sessionId', accessToken.access_token);
 return accessToken.access_token;
```